doc/specs/#117-Windows-Store-Support.md

```diff
@@ -0,0 +1,144 @@
+---
+author: <kevin> <larkin> <kevinlams/<kevinla@microsoft.com>
+created on: <2020-05-28>
+last updated: <2020-05-28>
+issue id: <github issue id>
+---
+# ===== DRAFT ======
+# Package Manager Windows Store Support
+
+## Abstract
+
+
+This spec identifies requirements and implementation for supporting Windows Store apps in the Windows Package Manager.
+
+## Inspiration
+
+
+The Windows Store is a curated list of applications.  The store apps include tools and the best of class applications and must be available to Package Manager users.
+
+## Solution Design
+
+Windows Package Manager will consume a manifest file like it does with all other formats.  The Windows Package Manager will then use a store API to initiate downloads and install the apps.
+
+
+
+**Store Apps**
+The Store has multiple app types.  Those that are free for download and free, but with purchase options, as well as paid apps.
+
+***Free Apps***  
+Apps that are free, will be downloaded directly when requested.
+
+***Pay apps [NOT SUPPORTED]***
+Apps that are for purchase, will not be allowed to be submitted.  If an app from the store requires commerce, to run, we will not allow them in the Windows Package Manager repository.  
```

Comment:
What about paid apps that have been purchased already?

Reply:
If the entitlement is already granted, I think we can continue with installation.